heron/spi/src/java/com/twitter/heron/spi/utils/TMasterUtils.java

```diff
@@ -65,48 +65,42 @@ public static boolean sendToTMaster(String command, String topologyName,
     try {
       URL endpoint = new URL(url);
       LOG.fine("HTTP URL for TMaster: " + endpoint);
-
-      return sendGetRequest(endpoint, command, tunnelConfig);
+      sendGetRequest(endpoint, command, tunnelConfig);
     } catch (MalformedURLException e) {
-      throw new RuntimeException("Invalid URL for tmaster endpoint: " + url, e);
+      throw new RuntimeException("Invalid URL for Tmaster endpoint: " + url, e);
```

Comment:
Can we use TMasterException instead of RuntimeException? Also the TM in TMaster should both be in caps.